src/fides/api/main.py

```diff
@@ -321,14 +320,10 @@ def read_other_paths(request: Request) -> Response:
     # check first if requested file exists (for frontend assets)
     path = request.path_params["catchall"]
 
-    # First search in the local (dev) build for for a matching route.
-    ui_file = match_route(get_local_file_map(), path)
```

Comment:
it seemed like `get_package_file_map` when running via `fides_env(dev)` returned `/fides/src/fides` while the `get_local_file_map` returned `src/fides` (relative path). the relative path caused problems when doing the comparison we do further down to see if it has the same parent as the admin UI path.

but since `get_package_file_map` already returns `/fides/src/fides` (or `/usr/local/lib/python3.10/site-packages/fides` for built packages) it seems like we can just use that in lieu of `get_local_file_map`? I may very well be missing something, but this seems to work